### PR TITLE
now counting user propensity fetch status for messages pertaining to the article page

### PR DESCRIPTION
**Pull request by ifyio**
cc/ @commuterjoy

### CODE REVIEW

**Comment by commuterjoy** [comment]
looks good +1

/cc @tom-parker

File: index.js [hunk 1]
```diff
@@ -49,8 +49,8 @@ var sqsUrlEgest = process.env.SQS_EGEST;
 
 					sink.pusher(parsedMessage.egest.annotations);
 					sink.redis(parsedMessage.egest.annotations);
-					sink.keen(parsedMessage);
```
Comment (tom-parker) [inline index.js]: Not sure if this was intentional?

**Comment by tom-parker** [comment]
Looks good, wondering if we can get an env var setup for this too e.g https://github.com/Financial-Times/spoor-consumer/blob/master/sinks/keen-redux.js#L34

We don't really use them at the moment but for consistency, all the other sinks have them at the moment

File: index.js (hunk 1 above)
Comment (ifyio) [inline index.js]: Wasn't intentional. Good spot.

**Comment by ifyio** [comment]
@tom-parker: have setup env vars. Are you happy with this?

**Comment by tom-parker** [comment]
:+1:  great thanks

**Pull request merged**